db/seeds.rb

```diff
@@ -1,43 +1,35 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
-
-Category.destroy_all
-Refinery::Authentication::Devise::User.destroy_all
-
-%w(Residential Institutional Mixed-Use Hospitality Commercial).each do |c|
-  Category.create(name:c)
-end
-
 user_data = [
-  {username:"Logan", email:"l@l.com", password: "password"},
-  {username:"Pavan", email:"p@p.com", password: "password"},
-  {username:"Arjun", email:"a@a.com", password: "password"},
-  {username:"Forefront", email:"f@f.com", password: "password"}
+  {username:"Forefront", email:"f@f.com", password: "password"},
```

Comment:
That would, however mean that we'd need them to make us accounts for us to be able to do anything on it.